lib/src/view/design_main_strand_dna_end.dart

```diff
@@ -159,6 +159,9 @@ class DesignMainDNAEndComponent extends UiComponent2<DesignMainDNAEndProps> with
     if (end_selectable(dna_end)) {
       // select end
       MouseEvent event = event_synthetic.nativeEvent;
+      if(event.button == 2){
```

Comment:
Can you replace the literal `2` with `constants.RIGHT_CLICK_BUTTON`? Then it's easier to tell what this line of code is doing.

Also can you test prevent middle-click from selecting as well? Currently we don't use middle-click for anything, but in the future we might, and this would help prevent the behaviors from colliding.